resources/views/calendar.blade.php

```diff
@@ -1,6 +1,22 @@
 <!DOCTYPE html>
+<<<<<<< Updated upstream
```

Comment:
There are some git stash merge issues going on here.  Anything with a <<<<<< needs to be resolved before being merged.